src/assets/css/_sass/elements/_navs.scss

```diff
@@ -3,3 +3,40 @@
   Navs
 
 */
+
+#nav-header {
```

Comment:
See comment on previous commit, use classes instead of ids, change selector to be `.site-header`